Buddies.API/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace Buddies.API
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+
+        public string? Summary { get; set; }
+    }
+}
```

Comment:
Perhaps remove some of these boilerplate files. Or at least rename them to something relevant and keep it empty.